sequencer/api/node_state.toml

```diff
@@ -0,0 +1,15 @@
+[route.version]
+PATH = ["/current_version"]
+METHOD = "GET"
+DOC = "Get the current version for the node."
+
+[route.chain_config]
+PATH = ["/chain_config"]
```

Comment:
shouldn't we return validated state chain config because node state's chain config is the base version's chain config